src/metatags/opportunity-utils.js

```diff
@@ -11,7 +11,13 @@
  */
 
 export function removeTrailingSlash(url) {
```

Comment:
Can we create a new function as getBaseUrl and use that? This might be used somewhere where we expect to only to remove the trailing slash.